### PR TITLE
Fi 3759 save bundles from credential strings

### PR DESCRIPTION
**Pull request by bmeshell**
# Summary
save fhir_bundles and use as input to the shc_fhir_validation test to avoid duplicating the work of decoding verifiable credential string

# Testing Guidance
run unit tests, run test kit in browser and test with health_cards reference server

### CODE REVIEW

**Comment by yunwwang** [comment]
Please address the merge conflict by rebasing this work branch to the latest main branch

File: lib/smart_health_cards_test_kit/shc_fhir_validation.rb [hunk 1]
```diff
@@ -7,33 +7,13 @@ class SHCFHIRValidation < Inferno::Test
...
-        raw_bundle = subject['fhirBundle']
-        assert raw_bundle.is_a?(Hash), "Expected 'vc.fhirBundle' to be a JSON object, but found #{raw_bundle.class}"
-
-        bundle = FHIR::Bundle.new(raw_bundle)
-        assert_valid_resource(resource: bundle)
-        bundle_array.append(bundle)
-
-      end
-      output fhir_bundles: bundle_array
+      #skip_if fhir_bundles.blank?, 'No FHIR bundles received'
```
Comment (yunwwang) [inline lib/smart_health_cards_test_kit/shc_fhir_validation.rb:14]: why is this line commented out?

File: spec/smart_health_cards/shc_fhir_validation_spec.rb [hunk 2]
```diff
@@ -29,31 +29,114 @@ def run(runnable, inputs = {})
...
       expect(result.result).to eq('pass')
     end
 
-    it 'raises an error if the JWS payload does not conform to the FHIR Bundle profile' do
-      credential_strings = 'asdf'
-      expect {result = run(test, { file_download_url: url, url: url, credential_strings: credential_strings })}.to raise_error()
+    it 'raises an error if the input does not conform to the FHIR Bundle profile' do
+      result = run(test, { file_download_url: url, url: url, fhir_bundles: 'asdf'})
+      expect(result.result).to eq('error')
     end
```
Comment (yunwwang) [inline spec/smart_health_cards/shc_fhir_validation_spec.rb]: add test for empty array

**Comment by bmeshell** [comment]
No need to merge this PR. It is superseded by [Pull request 8](https://github.com/inferno-framework/smart-health-cards-test-kit/pull/8) which includes all of the changes on this branch plus more.
